cpp/src/parquet/file_serialize_test.cc

```diff
@@ -330,8 +330,7 @@ TYPED_TEST(TestSerialize, SmallFileGzip) {
 
 #ifdef ARROW_WITH_LZ4
 TYPED_TEST(TestSerialize, SmallFileLz4) {
-  ASSERT_NO_FATAL_FAILURE(
-      this->FileSerializeTest(Compression::LZ4, Compression::LZ4_HADOOP));
+  ASSERT_NO_FATAL_FAILURE(this->FileSerializeTest(Compression::LZ4));
```

Comment:
shouldn't we start to fail with LZ4?  Or at least have a warning?

Reply:
Hmm, no, `Compression::LZ4` is exactly the LZ4 raw format.